src/main/java/net/endercube/gamelib/utils/ComponentUtils.java

```diff
@@ -51,110 +82,17 @@ public static Component convertToSmallCaps(Component input) {
     }
 
     /**
-     * @inheritDoc
+     * @see <a href="https://discord.com/channels/706185253441634317/1001092833677152397/1326883302455640104">Discord</a>
```

Comment:
Could this be credit: Discord instead of just Discord, to make it clear what the Discord link is for